providers/reamaze.go

```diff
@@ -0,0 +1,23 @@
+package providers
+
+const Reamaze Provider = "reamaze"
+
+func init() {
+	SetInfo(Reamaze, ProviderInfo{
+		AuthType: Basic,
+		BaseURL:  "https://{{.workspace}}.reamaze.io/api",
```

Comment:
We will need to shift `ExplicitWorkspaceRequired` out of `Oauth2Opts` before we can support this. Keeping this PR on hold until we can do that